### PR TITLE
AKS Script Action - Parameterize script for extensibility 

### PR DESCRIPTION
**Pull request by dciborow**
## Description

If you haven't already, read the full [contribution guide](../CONTRIBUTING.md). The guide may have changed since the last time you read it, so please double-check. Once you are done and ready to submit your PR, edit the PR description and run through the relevant checklist below.

## Adding a new module

- [ ] A proposal has been submitted and approved.
- [ ] I have included "Closes #{module_proposal_issue_number}" in the PR description.
- [ ] I have run `brm validate` locally to verify the module files.
- [ ] I have run deployment tests locally to ensure the module is deployable.

## Updating an existing module

- [ ] This is a bug fix:
  - [ ] Someone has opened a bug report issue, and I have included "Closes #{bug_report_issue_number}" in the PR description.
  - [ ] The bug was found by the module author, and no one has opened an issue to report it yet.
- [x] I have run `brm validate` locally to verify the module files.
- [x] I have run deployment tests locally to ensure the module is deployable.
- [x] I have read the [Updating an existing module](../CONTRIBUTING.md#updating-an-existing-module) section in the contributing guide and updated the `version.json` file properly:
  - [ ] The PR contains backwards compatible bug fixes, and I have NOT bumped the MAJOR or MINOR version in `version.json`.
  - [x] The PR contains backwards compatible feature updates, and I have bumped the MINOR version in `version.json`.
  - [ ] The PR contains breaking changes, and I have bumped the MAJOR version in `version.json`.

### CODE REVIEW

File: modules/deployment-scripts/aks-run-command/README.md [hunk 1]
```diff
@@ -26,100 +31,12 @@ An Azure CLI Deployment Script that allows you to run a command on a Kubernetes
...
-      rbacWriter
-    ]
-    commands: [
-      'kubectl run nginx --image=nginx'
-      'kubectl get pods'
-    ]
-  }
-}
-```
+### Example 1
```
Comment (Gordonby) [inline modules/deployment-scripts/aks-run-command/README.md:34]: The Readme Examples have been lost

File: modules/deployment-scripts/aks-run-command/README.md [hunk 2]
```diff
@@ -17,6 +21,7 @@ An Azure CLI Deployment Script that allows you to run a command on a Kubernetes
 | `commands`                                 | `array`  | Yes      | An array of commands to run                                                                                   |
 | `initialScriptDelay`                       | `string` | No       | A delay before the script import operation starts. Primarily to allow Azure AAD Role Assignments to propagate |
 | `cleanupPreference`                        | `string` | No       | When the script resource is cleaned up                                                                        |
+| `scriptContent`                            | `string` | No       | Default Script to issue commands to AKS                                                                       |
```
Comment (Gordonby) [inline modules/deployment-scripts/aks-run-command/README.md:24]: By allowing any script to be run in this module, (not just aks commands) doesn't that genericise it's capability a little too far?

What's the use case that this lights up?

Comment (dciborow) [reply modules/deployment-scripts/aks-run-command/README.md:24]: @Gordonby , if you check out the PR I have over here, https://github.com/Azure/azure-quickstart-templates/pull/12698, I have had to duplicate all the logic in this module. 
Generally, this lights up the ability to target AKS with Custom script actions.

In the current examples, there is one dedicated for HELM, but helm is not typically run on AKS, and instead targets AKS. This provides a more natural way to leverage helm with AKS.

(But I will give you its a bit of hack on what has been created)

File: modules/deployment-scripts/aks-run-command/README.md (hunk 1 above)
Comment (dciborow) [reply modules/deployment-scripts/aks-run-command/README.md:34]: @Gordonby , I can not figure out what happened here... "brm generate" will not produce new examples... 

I assume these should not be manually created?

**Pull request closed without merging**